features/old/menu/api_menu.feature

```diff
@@ -54,7 +54,7 @@ Feature: API menu
     | Settings                  |
 
   Scenario: Integration sub menu structure for API as Product
-    Given the service acts as product
+    Given the account has Service acting as Product
```

Comment:
I know this is intended to be readable for dummies, but who are kidding? It's just not anyway. IMHO the code would be easier to understand if it clearly stated what it does and in this case it has to do with a rolling update being enabled.

```suggestion
    Given the account has api_as_product rolling update enabled
```

If you don't agree, it's all right too, but then can we please describe it

```
Given the account uses API as product
```

or

```
Given the account has all services acting as products
```

?